packages/components/package.json

```diff
@@ -10,33 +10,32 @@
   "homepage": "https://github.com/chanzuckerberg/edu-design-system/tree/main/packages/components",
   "license": "MIT",
   "main": "dist/index.js",
-  "publishConfig": {
-    "access": "public",
-    "directory": "dist/",
-    "registry": "https://registry.npmjs.org/"
-  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/chanzuckerberg/edu-design-system.git"
   },
+  "files": [
+    "dist",
+    "tailwind.config.js"
+  ],
```

Comment:
This tells npm to only publish files from `dist` and the `tailwind.config.js` file. Everything else (such as src/) will be ignored, except for some files that are always included in the published package (package.json, readme, changelog, etc).

Reply:
An alternative would be a .npmignore file listing the src/, .storybook/, coverage/, and storybook-static/ directories.